core/kazoo_amqp/src/api/kapi_acdc_queue.erl

```diff
@@ -60,7 +60,7 @@
 
 -export([queue_size/2, shared_queue_name/2]).
 
--include("acdc.hrl").
+-include("../../applications/acdc/src/acdc.hrl").
```

Comment:
the problem is that crossbar acdc modules use kapi_acdc but now we have deployments that have only crossbar, thus kapi_acdc is not installed.

this brings back the discussion on queue/agent api abstraction and having apps handling them, acdc or qubicle or other, which is my preferred solution.